edgexfoundry/device-rfid-llrp-go
# refactor: Remove code that loads static provision watchers

closes: #128

Signed-off-by: Marc-Philippe Fuller <marc-philippe.fuller@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rfid-llrp-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:


## Testing Instructions

1. Run `make test` to make sure no unit tests are broken
2. bring up edgex stack running no-secty and as-llrp
3. `make build`
4. `cd cmd`
5. `./device-rfid-llrp`
6.  looks through the logs to verify I see "Loading pre-defined provision watchers "

## New Dependency Instructions (If applicable)
